src/components/About.jsx

```diff
@@ -0,0 +1,36 @@
+import "../blocks/About.css";
+import authorImg from "../assets/images/me.jpg";
+
+export default function About() {
+  return (
+    <section className="about">
+      <img src={authorImg} alt="image of the author" className="about__image" />
+      <div className="about__text-content">
+        <h2 className="about__text-title">About the author</h2>
+        <p className="about__text">
+          Hello! Thank you for taking a moment to read my About section.
+          <br />
```

Comment:
` <br />` should not be used in the markup because it's not a good practice. Please, make a `section` in line 10 which will contain 4 `p` tags . This way you'll have 4 paragraphs